webcompat/static/css/development/components/button.css

```diff
@@ -26,6 +26,7 @@
   background-image: none;
   user-select: none;
   -webkit-appearance: none;
+  margin: 0 1em .5em 1em;
```

Comment:
@magsout this should address your comment.

https://github.com/webcompat/webcompat.com/pull/1265/commits/e7902a465c66e155fdf9f63ef0ce3233585cad49